src/hdmf/backends/io.py

```diff
@@ -46,6 +50,9 @@ def read(self, **kwargs):
             # TODO also check that the keys are appropriate. print a better error message
             raise UnsupportedOperation('Cannot build data. There are no values.')
         container = self.__manager.construct(f_builder)
+        if self.external_resources_path is not None:
+            from hdmf.common import ExternalResources
+            self.external_resources = ExternalResources.from_norm_tsv(path=self.external_resources_path)
```

Comment:
Should the `ExternalResources` object not also be linked to the container if possible? Something along the lines of:

```suggestion
            self.external_resources = ExternalResources.from_norm_tsv(path=self.external_resources_path)
            if isinstance(container, ExternalResourcesManager):
                container.link_resources(external_resources=self.external_resources)
```

Reply:
Possibly also raise a warning if the `ExternalResources` was not linked to the Container.

Reply:
Which approach are we using as default for saving the ExternalResources?